infoschema/error.go

```diff
@@ -69,4 +69,6 @@ var (
 	ErrTableLocked = dbterror.ClassSchema.NewStd(mysql.ErrTableLocked)
 	// ErrWrongObject returns when the table/view/sequence is not the expected object.
 	ErrWrongObject = dbterror.ClassSchema.NewStd(mysql.ErrWrongObject)
+	// ErrAdminCheckTable returns when the check table in temporary mode
+	ErrAdminCheckTable = dbterror.ClassSchema.NewStd(mysql.ErrAdminCheckTable)
```

Comment:
I don't think it's suitable to put this error in `infoschema`. I think reusing `executor.ErrAdminCheckTable` is fine.

Reply:
but if i use `executor.ErrAdminCheckTable`, it would have an import cycle